modules/ROOT/pages/platform/logging.adoc

```diff
@@ -13,7 +13,12 @@ To access the *Logs* tab:
 
 == Query logs
 
-A query log provides a log of queries executed on an instance within a specified time range. 
+A query log provides a log of queries executed on an instance within a specified time range.
+
+[NOTE]
+====
+Queries that complete succesfully within 50ms are not logged.
```

Comment:
```suggestion
Queries that complete successfully within 50ms are not logged.
```

Reply:
how grammarly doesn't see this as a spelling mistake I have no fucking idea.